roles/netbootxyz/templates/menu/utils-efi.ipxe.j2

```diff
@@ -17,8 +17,8 @@ goto ${menu} ||
 chain ${menu}.ipxe || goto utils_exit
 goto utils_exit
 
-{% for key, value in utilities.items() | sort %}
-{% if value.enabled | bool and value.menu == "efi" %}
+{% for key, value in utilitiesefi.items() | sort %}
+{% if value.enabled | bool and value.type == "memdisk" %}
```

Comment:
Probably won't need the memdisk type check if it's EFI.

Reply:
That logic specifically guards the memdisk menu entries and  allows the core one to boot from the clonezilla.ipxe file.

Reply:
Yeah, just saying we can probably remove all of the memdisk stuff from utils-efi.ipxe.j2 while we're at it because it won't ever work in efi mode.

Reply:
I concur you want it in the scope of this PR?

Reply:
I'll merge this one in, and we can clean up in another.